ci-operator/step-registry/ipi/gcp/private/pre/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
The same suggestion as before, to avoid affecting from the potential OWNERS change of the parent folder, and keep the same style as other platforms.